validphys2/src/validphys/config.py

```diff
@@ -477,6 +477,33 @@ def parse_dataset_input(self, dataset: Mapping):
             variant=variant,
         )
 
+    def parse_inconsistent_data_settings(self, settings):
+        """
+        Parse the inconsistent data settings from the yaml file.
+        """
+        known_keys = {
+            "treatment_names",
+            "names_uncertainties",
+            "inconsistent_datasets",
+            "sys_rescaling_factor",
+        }
+
+        kdiff = settings.keys() - known_keys
+        for k in kdiff:
+            log.warning(
+                ConfigError(f"Key '{k}' in inconsistent_data_settings not known.", k, known_keys)
+            )
```

Comment:
I think it is better to raise an exception here. By being polite in the past we have ended up with runcards that have useless options in random places. Better to be strict.

```python
        if kdiff:
            raise ConfigError(f"Bad keys: {kdiff}")
```